Client/Frontend/Browser/BrowserScrollController.swift

```diff
@@ -66,7 +66,7 @@ class BrowserScrollingController: NSObject {
     private var contentSize: CGSize { return scrollView?.contentSize ?? CGSizeZero }
     private var scrollViewHeight: CGFloat { return scrollView?.frame.height ?? 0 }
     private var headerFrame: CGRect { return header?.frame ?? CGRectZero }
-    private var footerFrame: CGRect { return footer?.frame ?? CGRectZero }
+    private var footerFrame: CGRect { return urlBar?.frame ?? CGRectZero }
```

Comment:
Might want to rename this property now that it's not using the footer's frame but the urlBar one instead. I notice that we only really want the height and don't care if it's a rect. Maybe rename this to bottomScrollHeight and the header one to topScrollHeight?